### DISCUSSION TITLE
sidebar / scroll-sidebar for navbar?

### DISCUSSION THREAD

**Discussion by DanielUPedersen1** [twbs/bootstrap#36355] [Ideas] [answered]
Hello, Daniel here. Been a while online but slowly getting back into the game. Big fan of your project and just seen your latest release! Well done so far! :D

However, i do have a suggestion. I did make something of similar before but was on another github account a little while back (got hacked sadly).

Anywho, as the title says. I'm a big fan of building with the help of PHP projects in my sparetime! One of which, includes Administrator panels/dashboards and such. However i do miss the option or choice (if you can call it that), to actually "implement" code easier to build stuff like vertical navbars (accordion menus i suppose you could call them?).

My ideas are always the same one way or another but in different approaches. Sidebars. Whenever a user-account is logged onto admin dashboard, they have the main navigational bar at top or sticky bottom, and then i besides cards for that magical flow of options from various tools and such, i wish there were the option to insert a kinda "scrool-looking" sidebar that would act as a sidebar/side-navigational navbar kinda thingy. I always find myself adding or designing those on my own by hand (sadly).

So just an idea for a possible V5.3 release if possible, i will even help design the thing if you guys could need my help? I do CSS only tho, not a big fan of SCSS (i can do it but been a long time since and i mostly deal with regular basic css styling for idea purposes really :P)

Anywho, i hope you will strongly consider my proposal and add it at a later point!

Thanks for the time and keep up the great work! Again, a big fan and i will always follow you guys and just say the word if you need a suppliment for designer... im always at the PC when im not at sea :D

Good day/night.

// DUP ;)

**Comment by mdo** [comment, marked as answer]
- https://getbootstrap.com/docs/5.2/components/accordion/
- https://getbootstrap.com/docs/5.2/components/navbar/#responsive-behaviors
- https://getbootstrap.com/docs/5.2/components/offcanvas/
- https://getbootstrap.com/docs/5.2/examples/sidebars/
- https://getbootstrap.com/docs/5.2/examples/dashboard/

**Reply by DanielUPedersen1** [reply]
oh dang it, i forgot all about this my bad! Thanks for the answer tho :D
